src/errors/error_handler.js

```diff
@@ -0,0 +1,61 @@
+import RedirectError from './API/redirect_error';
+import UnauthorizedError from './API/unauthorized_error';
+import UnknownApiError from './API/unknown_api_error';
+import PaymentError from './API/payment_error';
+import NotFoundError from './API/not_found_error';
+import MethodNotAllowedError from './API/method_not_allowed_error';
+import TimeoutError from './API/timeout_error';
+import InvalidRequestError from './API/invalid_request_error';
+import RateLimitError from './API/rate_limit_error';
+import InternalServerError from './API/internal_server_error';
+import ServiceUnavailableError from './API/service_unavailable_error';
+import GatewayTimeoutError from './API/gateway_timeout_error';
+import ForbiddenError from './API/forbidden_error';
+
+export default class ErrorHandler {
+  static handleError(error) {
```

Comment:
It should be noted that this only handles API specific errors, please clarify the function name and add a docstring

```suggestion
  static handleApiError(error) {
```